src/App.js

```diff
@@ -1,26 +1,21 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+import {  Route } from 'react-router-dom';
+import HomePage from './components/pages/HomePage';
+import LoginPage from './components/pages/LoginPage';
+import AboutPage from './components/pages/AboutPage';
+import'semantic-ui-css/semantic.min.css';
+const App = () => (
+    <div class = "ui secondary menu">
+      <a class = "active item"
```

Comment:
Use `<Link>` or `<NavLink>` instead of `<a>` when linking to other pages. https://reactrouter.com/web/api/Link

Reply:
Fixed